golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -774,6 +774,268 @@
         "age": "YearsUpto14"
       }
     },
+    "dc/p/119gy3zecnm61": {
```

Comment:
@beets regarding your question -- yes we were expecting diffs here. I agree that this is very difficult to look at. I will take a closer look on staging if that's ok.